README.md

```diff
@@ -224,7 +224,8 @@ The following subcommands are supported:
                          dumped.
 
                          Syntax: v8 bt [number]
-      findjsinstances -- List all objects which share the specified map.
+      findjsinstances -- List every object with the specified type name.
+                         Use -i or --inspect to display detailed `v8 inspect` outputfor each object.
```

Comment:
Good spot, it was in the help string in llnode.cc so I've corrected both places.